config/channels/in-memory-channel/300-in-memory-channel.yaml

```
@@ -64,10 +49,28 @@ spec:
     - name: Age
       type: date
       JSONPath: .metadata.creationTimestamp
-  versions:
   - name: v1alpha1
     served: false
     storage: false
-  - name: v1beta1
-    served: true
-    storage: true
+    subresources: *subresources
+    schema: *schema
+    additionalPrinterColumns: *additionalPrinterColumns
+  names:
+    kind: InMemoryChannel
+    plural: inmemorychannels
+    singular: inmemorychannel
+    categories:
+    - all
+    - knative
+    - messaging
+    - channel
+    shortNames:
+    - imc
+  scope: Namespaced
+  conversion:
+    strategy: Webhook
+      conversionReviewVersions: ["v1beta1", "v1alpha1"]
```

These versions are the payload being set to the webhook - not the CRD version being reviewed

```suggestion
      conversionReviewVersions: ["v1", "v1beta1"]
```

---

We need both v1 and v1beta1 because our min K8s version is 1.16.

The API imposes a restriction that we support conversionReview version that are available on K8s 1.15 & 1.16.

We can't drop v1beta1 it was the only available version in 1.15